src/app/validators/company.async-validations.ts

```diff
@@ -1,53 +1,78 @@
-import { create, enforce, group, only, test } from 'vest';
+import { create, enforce, group, only, test, omitWhen } from 'vest';
 
 import { AppValidationContext } from './app-validation-context';
 import { Company } from '@model';
-import { FeinValidationResponse, isGoodFein } from '@services/fein-validation.service';
-import { ValidationContext, ValidationSuite, ValidationSuiteFn } from '@validation';
+import {
+  FeinValidationResponse,
+  isGoodFein,
+} from '@services/fein-validation.service';
+import {
+  ValidationContext,
+  ValidationSuite,
+  ValidationSuiteFn,
+} from '@validation';
 
 export function createCompanyAsyncValidationSuite() {
-  return create('CompanyAsyncValidationSuite',
-    (model: Partial<Company>, field?: string, groupName?: string, context?: ValidationContext) => {
-    only(field); // if field defined, limit to tests of this field
-    only.group(groupName); // if groupName defined, limit to tests of this group
-    group('company', () => companyAsyncValidations(model, field, groupName, context));
-  }) as ValidationSuite;
+  return create(
+    'CompanyAsyncValidationSuite',
+    (
+      model: Partial<Company>,
+      field?: string,
+      groupName?: string,
+      context?: ValidationContext
+    ) => {
+      only(field); // if field defined, limit to tests of this field
+      only.group(groupName); // if groupName defined, limit to tests of this group
+      group('company', () =>
+        companyAsyncValidations(model, field, groupName, context)
+      );
+    }
+  ) as ValidationSuite;
 }
 
-export const companyAsyncValidationSuite: ValidationSuite = createCompanyAsyncValidationSuite();
+export const companyAsyncValidationSuite: ValidationSuite =
+  createCompanyAsyncValidationSuite();
 
 /** Company Asynchronous Validation Suite */
-export const companyAsyncValidations: ValidationSuiteFn =
-  (model: Partial<Company>, field?: string, groupName?: string, context?: ValidationContext) => {
-    model = model ?? {};
+export const companyAsyncValidations: ValidationSuiteFn = (
+  model: Partial<Company>,
+  field?: string,
+  groupName?: string,
+  context?: ValidationContext
+) => {
+  model = model ?? {};
 
-    const fvs = (<AppValidationContext>context)?.feinValidationService;
-    /** FEIN check function (or FN returning "not found" result) */
-    const checkFein = fvs
-      ? fvs.check.bind(fvs)
-      // No fvs? This is almost certainly a bug. Remember to provide a context.
-      : (fein: string) => Promise.resolve<FeinValidationResponse>({ fein });
+  const fvs = (<AppValidationContext>context)?.feinValidationService;
+  /** FEIN check function (or FN returning "not found" result) */
+  const checkFein = fvs
+    ? fvs.check.bind(fvs)
+    : // No fvs? This is almost certainly a bug. Remember to provide a context.
+      (fein: string) => Promise.resolve<FeinValidationResponse>({ fein });
 
+  omitWhen(!isGoodFein(model.fein), () => {
```

Comment:
omitWhen allows these tests to be skipped completely, without marking the suite as invalid.
https://vestjs.dev/docs/writing_your_suite/including_and_excluding/omitWhen